OpenRA.Mods.Common/OpenRA.Mods.Common.csproj

```diff
@@ -377,10 +385,23 @@
     <Compile Include="Widgets\Logic\ButtonTooltipLogic.cs" />
     <Compile Include="Widgets\Logic\ColorPickerLogic.cs" />
     <Compile Include="Widgets\Logic\CountryTooltipLogic.cs" />
-    <Compile Include="Widgets\Logic\DisconnectWatcherLogic.cs" />
     <Compile Include="Widgets\Logic\Ingame\IngameRadarDisplayLogic.cs" />
     <Compile Include="Widgets\Logic\Ingame\LoadIngamePlayerOrObserverUILogic.cs" />
     <Compile Include="Widgets\Logic\ModBrowserLogic.cs" />
+    <Compile Include="Widgets\Logic\SettingsLogic.cs" />
+    <Compile Include="Widgets\Logic\TabCompletionLogic.cs" />
+    <Compile Include="Widgets\Logic\Networking\ClientTooltipLogic.cs" />
+    <Compile Include="Widgets\Logic\Networking\ConnectionLogic.cs" />
+    <Compile Include="Widgets\Logic\Networking\DirectConnectLogic.cs" />
+    <Compile Include="Widgets\Logic\Networking\DisconnectWatcherLogic.cs" />
+    <Compile Include="Widgets\Logic\Networking\KickClientLogic.cs" />
+    <Compile Include="Widgets\Logic\Networking\KickSpectatorsLogic.cs" />
+    <Compile Include="Widgets\Logic\Networking\LobbyLogic.cs" />
```

Comment:
I'm not convinced that all of these really belong in Networking.  If you want to sub-group these, then `ClientTooltipLogic` and `LobbyLogic`, `LobbyMapPreviewLogic`, `LobbyUtils` would fit better in their own Lobby directory.